zookeeper/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM fedora:26
+
+RUN dnf -y install java-1.8.0-openjdk gettext hostname && dnf clean all -y
+
+# set Kafka home folder
+ENV ZOOKEEPER_HOME=/opt/zookeeper
+
+# Add zookeeper group / user
+RUN groupadd -r -g 1001 zookeeper && useradd -r -m -u 1001 -g zookeeper zookeeper
+
+# Set Zookeeper version
+ENV ZOOKEEPER_VERSION=3.4.10
+
+# Set MD5checksum
+ENV CHECKSUM="e4cf1b1593ca870bf1c7a75188f09678  zookeeper-3.4.10.tar.gz"
+
+# Set from build args
+ARG version=latest
+ENV VERSION ${version}
+
+WORKDIR $ZOOKEEPER_HOME
+
+# downloading/extracting Apache Zookeeper
+RUN curl -O http://www.eu.apache.org/dist/zookeeper/zookeeper-$ZOOKEEPER_VERSION/zookeeper-$ZOOKEEPER_VERSION.tar.gz  \
```

Comment:
Is it downloadable via HTTPS? (I know we're checksumming what we download, but it's only using MD5 and some clever [collision attacks](http://www.mathstat.dal.ca/~selinger/md5collision/) are known).

Reply:
Moves to SSL download links and stronger checksum algorithms (SHA1 for Zookeeper and SHA512 for Kafka).